Sources/PointFree/Account/Account.swift

```diff
@@ -56,12 +56,9 @@ private func fetchAccountData<I, A>(
       .map { $0.right ?? [] },
 
     AppEnvironment.current.database.fetchEmailSettingsForUserId(user.id).run.parallel
-      .map { $0.right ?? [] },
-
-    AppEnvironment.current.database.fetchEpisodeCredits(user.id).run.parallel
```

Comment:
very strange that a `zip5` crashes on linux. my work to figure out this crash is here https://github.com/pointfreeco/pointfreeco/compare/zip-crash